src/main/java/net/sf/rails/game/model/PortfolioModel.java

```diff
@@ -245,9 +247,9 @@ public int getShare(PublicCompany company) {
 
 
     /**
-     * @return the number of shares owned by the PorfolioModel for this company
+     * @return the number of shares owned by the PortfolioModel for this company
      */
-    public int getShareNumber(PublicCompany company) {
+    public int getShares(PublicCompany company) {
```

Comment:
This name seems misleading to me. This method does not return the shared but the number of owned shares. I.e. I think it should be named something like `getNumberOfShares` right?

Reply:
There are no share objects (but certificates), so you can't get any. Of course, what you propose is more unambiguous, but I consider brevity as a worthy goal too.

I tend to use the following nomenclature:
Share = a percentage of company ownership,
Shares = number of share units (usually 5% or 10%, sometimes more),
Cert = a single share certificate,
Certs = a collection of share certificates.